integration-tests/Utils.js

```diff
@@ -159,7 +159,9 @@ const extractSummary = (stdout: string) => {
 // unifies their output to make it possible to snapshot them.
 // TODO: Remove when we drop support for node 4
 const cleanupStackTrace = (output: string) => {
-  return output.replace(/^.*at.*[\s][\(]?(\S*\:\d*\:\d*).*$/gm, '      at $1');
+  return output
+    .replace(/.*(?=packages)/g, '      at ')
```

Comment:
needed?

Reply:
@SimenB , that's to truncate the overly long relative path that jest-message-util generates between the ```__test__``` folder and jest itself when it runs the integration tests for ```global.test.js``` . Looks like this on my machine with it:

```packages/jest-jasmine2/build/jasmine/Env.js:433:15```

and like this without it:

```../../../../../../../Users/brian/Documents/projects/opensource/jest/packages/jest-jasmine2/build/jasmine/Env.js:4```

I can add a comment in there explaining if you think it warrants it? Or try a different approach if you'd rather.